src/pages/home/owners/Owners.tsx

```diff
@@ -1,16 +1,76 @@
-import { FunctionComponent } from "react";
+import { FunctionComponent, useState } from "react";
 import { Box } from "@mui/material";
-import { Typography } from "elements";
+import { TextInput, Typography } from "elements";
+import SearchRoundedIcon from "@mui/icons-material/SearchRounded";
+import theme from "theme";
+import OwnersTable from "./OwnersTable";
+import mockOwnersData from "./mockOwnersData";
+import NoOwnersYet from "./NoOwnersYet";
 
 const Owners: FunctionComponent = () => {
+  const ownersData = mockOwnersData; // temporary until API is ready
+
+  const [filteredData, setFilteredData] = useState(ownersData);
+  const [query, setQuery] = useState("");
+
+  const requestSearch = (searched: string) => {
+    setQuery(searched);
+    if (searched == "") {
+      setFilteredData(ownersData);
+    } else {
+      setFilteredData(
+        ownersData.filter(
+          (owner) =>
+            owner.name.toLowerCase().includes(searched.toLowerCase()) ||
+            owner.song.toLowerCase().includes(searched.toLowerCase())
+        )
+      );
+    }
+  };
+
   return (
     <Box
+      padding="60px"
       display="flex"
+      flexDirection="column"
       flexGrow={ 1 }
-      justifyContent="center"
-      alignItems="center"
+      flexWrap="nowrap"
+      justifyContent="flex-start"
+      alignItems="flex-start"
     >
-      <Typography>Owners page</Typography>
+      <Typography sx={ { pb: 4 } } variant="h1">
+        OWNERS
+      </Typography>
+
+      { ownersData.length ? (
+        <>
+          <Box sx={ { width: { xs: 200, sm: 380 }, pb: 3 } }>
+            <TextInput
+              value={ query }
+              onChange={ (e) => requestSearch(e.target.value) }
+              style={ {
+                width: "380px",
```

Comment:
I talked to tomas and he mentioned having search inputs be max width of 340px to stay consistent throughout our app.... I believe you can set the width to 100% here and the text input will take care of the max width since it is defined in the text-input component
if not, you can use the `maxWidth` from the theme.inputField

Reply:
Makes sense to me. I went ahead and changed it to 340px but the design specifies a width of 380